src/Components/TMS/Organisms/FOTMSDashboardNavbar/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import type { FC } from "react";
+// import Router from "next/router";
```

Comment:
please remove this comment

Reply:
this PR is only for header. everything under TMS folder is relatively for my note purpose and will be deleted in future usage.